packages/debug-contracts/src/styles.css

```diff
@@ -0,0 +1,163 @@
+@import "tailwindcss";
+
+@custom-variant dark (&:where(.dark, .dark *, [data-theme=dark], [data-theme=dark] *));
+
+@theme {
+  /* Debug Contracts Light Theme */
+  /* Core semantic colors */
+  --color-dc-primary: #93bbfb;
+  --color-dc-primary-content: #212638;
+  --color-dc-secondary: #dae8ff;
+  --color-dc-secondary-content: #212638;
+  --color-dc-accent: #93bbfb;
+  --color-dc-accent-content: #212638;
+  --color-dc-neutral: #212638;
+  --color-dc-neutral-content: #ffffff;
+
+  /* Base background colors */
+  --color-dc-base-100: #ffffff;
+  --color-dc-base-200: #f4f8ff;
+  --color-dc-base-300: #dae8ff;
+  --color-dc-base-content: #212638;
+
+  /* Status colors */
+  --color-dc-info: #93bbfb;
+  --color-dc-success: #34eeb6;
+  --color-dc-warning: #ffcf72;
+  --color-dc-error: #ff8863;
+
+  /* Utility colors */
+  --shadow-center: 0 0 12px -2px rgb(0 0 0 / 0.05);
+  --animate-pulse-fast: pulse 1s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+
+  /* Semantic class mappings */
+  --color-primary: var(--color-dc-primary);
```

Comment:
Let's say I have my app with my theme. I think if I'll import `debug-contracts` package and then import this css files, its colors and according classNames `bg-primary`, `border-primary` etc will overwrite my default colors and classNames

So I think there's two ways
1. Use default app primary and dc/sui as a fallback `--color-primary: var(--color-primary, --color-dc-primary);` (repeated `--color-primary`, hope it works lol)
2. Just use `-sui` variables which could be overwritten if needed at the parent div level or app global css level

Reply:
Updated at [`b4a9132` (#13)](https://github.com/scaffold-eth/scaffold-ui/pull/13/commits/b4a9132aa312c80d997bf4160f7a6e731b26f48c) and [`c1e83fb` (#13)](https://github.com/scaffold-eth/scaffold-ui/pull/13/commits/c1e83fb7ad67b3bbacfb7449f4d096ccb392b34e)